### DISCUSSION TITLE
Stop current progress and continue later (in Google Colab)

### DISCUSSION THREAD

**Discussion by ghost** [akai-katto/dandere2x#175] [Ideas]
As I was looking for Google Colab and looked on how to create, manage etc your notebooks, I saw that the free option only lasts for 12 hours on one go. So what I asked myself is: Is it possible to stop the current progress already made in Dandere, save it, and then start it again but with all the progess saved, so that you don't need to wait for these already processed frames?

If not, then I would love to have such an option. Would be useful for local runtimes and for Google Colab runtimes.

**Comment by akai-katto** [comment]
This was originally a feature, but due to the old dandere2x architecture, it was incredibly difficult to maintain, so I removed it overtime. I've moved to a new architecture which makes dandere2x easier to maintain + have features like this, so I'll re-consider adding it given that it's easier to work on, as well as people wanting to use the cloud for dandere2x more.

**Reply by ghost** [reply]
Would be definitely a really helpful and nice to have feature, especially for videos which take a long time to process.
